### DISCUSSION TITLE
Text edit widgets hover behaviour

### DISCUSSION THREAD

**Discussion by DavidColson** [emilk/egui#180] [Q&A] [answered]
Been testing out egui for a project, but I noticed that single and multiline text edit windows have this behaviour where hovering the text moves a little text edit cursor around as you hover? Is there a way to turn that off?

**Comment by emilk** [comment, marked as answer]
It is just a previous of where the cursor would go if you would click. There is no way to turn it off, but maybe it is more annoying than useful. I can add an option to turn it off, and have it off by default. Open an issue for it (or a PR)!

**Reply by emilk** [reply]
Turned off in https://github.com/emilk/egui/commit/834078a476329fbc9f16a6a436223cc752abeb48
